kde-base/legacy-icons/legacy-icons-4.11.22.ebuild

```diff
@@ -15,7 +15,8 @@ IUSE=""
 DEPEND=""
 RDEPEND="
 	!<kde-base/systemsettings-4.11.22-r1:4
-	!>=kde-frameworks/oxygen-icons-5.19.0:5
+	!=kde-frameworks/oxygen-icons-5.19.0:5
+	!=kde-frameworks/oxygen-icons-5.20.0:5
```

Comment:
upstream changed paths in oxygen-icons-5.21.0 to avoid even more collisions as new ones went in.